### ISSUE TITLE
Installation script failed due to python dependency 

### ISSUE THREAD

**Issue by technekey** [lyrebird-voice-changer/lyrebird#106]
I am running an Ubuntu machine with python3.10, however the ````install.sh```` is failing due to python version,
````
bash -x ./install.sh 
+ VERSION=1.1
+ VERBOSE=1
+ DRYRUN=0
+ '[' 0 = 1 ']'
+ ECHO_PREFIX='[lyrebird]'
++ id -u
+ '[' 1000 -eq 0 ']'
+ INSTALL_PREFIX=/home/ps/.local
+ verbose_echo 'Installing Lyrebird to prefix: /home/ps/.local'
+ '[' 1 = 1 ']'
+ echo '[lyrebird] Installing Lyrebird to prefix: /home/ps/.local'
[lyrebird] Installing Lyrebird to prefix: /home/ps/.local
+ BIN_PATH=/home/ps/.local/bin
+ SHARE_PATH=/home/ps/.local/share/lyrebird
+ DESKTOP_PATH=/home/ps/.local/share/applications
+ python_version_check
++ python3 --version
++ grep -Po '3\.\d'
+ PYTHON_VERSION=3.1
+ PYTHON_MIN_MAJOR=3
+ PYTHON_MIN_MINOR=7
+ PYTHON_MAJOR=3
+ PYTHON_MINOR=1
+ '[' 3 -lt 3 ']'
+ '[' 3 -eq 3 ']'
+ '[' 1 -lt 7 ']'
+ invalid_python
+ info_echo 'Lyrebird requires Python version 3.7 or higher'
+ echo '[lyrebird] Lyrebird requires Python version 3.7 or higher'
[lyrebird] Lyrebird requires Python version 3.7 or higher
+ exit 1
````
Here you can see my python version:

````
python3 --version
Python 3.10.4
````

The problem appears to be in the following code:

```
python_version_check() {
    PYTHON_VERSION=$(python3 --version | grep -Po '3\.\d')

    PYTHON_MIN_MAJOR=3
    PYTHON_MIN_MINOR=7

    PYTHON_MAJOR=${PYTHON_VERSION%.*}
    PYTHON_MINOR=${PYTHON_VERSION#*.}


```

Specifically in the following line:

`PYTHON_VERSION=$(python3 --version | grep -Po '3\.\d')
`

If the above line is changed to the following, the problem will get fixed:(````\d```` is changed to ````\d+````)

`PYTHON_VERSION=$(python3 --version | grep -Po '3\.\d+')
`

The version is only considering the single-digit minor version.

**Comment by ssieb** [comment]
Then you need to install the python3 gobject package.

**Comment by johnblommers** [comment]
Thanks! Indeed these steps are required in order to enable launch of the Lyrebird GUI:

```bash
cd /bin
sudo ln -s gcc gcc-5             # Because pycairo wants it
python3 -m pip install pycairo   # Because pygobject wants it
python3 -m pip install pygobject # Because lyrebird wants it
```

**Comment by dieek** [comment]
Just saw that I also ran into the same issue with Python 3.10.  This addition to the python version check worked for me on Fedora.  No other issues with finishing installation.

**Comment by harrego** [comment]
The >= Python 3.10 installer bug is fixed in the `main` branch and will be included in v1.1.1
